encodings/caesar/caesar_args.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+
+# Constants for modes
+brute_force = 'brute_force'
+word = 'word'
+letter = 'letter'
+
+def parse_args(input_args):
```

Comment:
I'm not sure if you're still investing much in the command line interface much. If so, I'd say
1. You can use "actions" to trigger desired operations after successful parsing.
1. For dealing with the `mode` parameter, you can use "subparsers" from `argparse` which allows you to use different actions for different `mode`s and to reuse arguments across different subparsers.

Although I'm not sure how much these code paths will be exercised, especially when the future plan is to make a web app out of this. So I'm ok with the parser code as it is.

Reply:
Those are great tips, but I think I'm sold on your idea of making this all run primarily in interactive mode. I'll update this accordingly